README.md

```diff
@@ -0,0 +1,13 @@
+# Repo for SRH_Jamaica
+
+## Setup
+1. Install python `>=3.6`
+2. Run `pip install -r requirements.txt`
+3. Run `npm install`
```

Comment:
Node/npm must be installed first, and for that we should recommend a version. My recommendation is latest LTS (Long-term support) version.